services/src/main/java/org/fao/geonet/guiservices/metadata/Sitemap.java

```diff
@@ -114,31 +113,33 @@ public Element exec(Element params, ServiceContext context) throws Exception {
 
         Element result = null;
 
-        // Requesting a sitemap specific document
         if (doc > 0) {
+            // Requesting a sitemap specific document
             if (doc <= pages) {
-                final PageRequest pageRequest = new PageRequest(doc - 1, _maxItemsPage, sortByChangeDateDesc);
 
-                result = metadataRepository.findAllAsXml(MetadataSpecs.hasMetadataIdIn(list), pageRequest);
+                final PageRequest pageRequest = new PageRequest(doc - 1, _maxItemsPage, sortByChangeDateDesc);
+                result = metadataRepository.findAllUuidsAndChangeDatesAndSchemaId(list, pageRequest);
```

Comment:
Sitemap should also exlcude templates & subtemplates. Could you add this filter too ie. metadata type = metadata only ?

Reply:
Done, thank you